packages/blobstorage/.gitignore

```diff
@@ -19,3 +19,4 @@
 
 # Go workspace file
 go.work
+.vercel
```

Comment:
we can just add this to the root gitignore because other vercel projects will benefit from it.